server/taskflows/hpccloud/taskflow/openfoam/DockerOpenFOAM_run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+cd $1
+
+# Env setup
+source /opt/OpenFOAM/OpenFOAM-v1612+/etc/bashrc
```

Comment:
Not really that's the docker path. For a real OpenFOAM deployment such that we can use MPI, more configuration will be needed. But not here.